src/workerd/server/server.c++

```diff
@@ -2429,25 +2429,24 @@ void Server::startServices(jsg::V8System& v8System, config::Config::Reader confi
                            kj::ForkedPromise<void>& forkedDrainWhen) {
   // ---------------------------------------------------------------------------
   // Configure inspector.
+  static constexpr uint DEFAULT_PORT = 9229;
+
+  static auto const listen = [](auto& network, auto& inspector, auto& inspectorService)
```

Comment:
I'd prefer to avoid `auto`-typed parameters if at all possible. Can we use lambda captures with kj::coCapture instead?

Same argument as here: https://github.com/cloudflare/workerd/pull/890#discussion_r1272590932

Reply:
I'll change from auto to explicitly typed. I'd actually prefer to avoid using `kj::coCapture` where it is not definitely necessary.